docs/en/operation/Journal.md

```diff
@@ -6,55 +6,57 @@ group: Operations
-- Talk about log files and checkpoints, how to manage the size
-
 * Table of Contents
 {:toc}
 
-Alluxio maintains a journal to support persistence of metadata operations.
-When a request modifies Alluxio state, e.g. creating or renaming files, the
-Alluxio master will write a journal entry for the operation before returning
-a successful response to the client. The journal entry is written to a
-persistent storage such as disk or HDFS, so even if the Alluxio
-master process is killed, state will be recovered on restart.
+Alluxio is fault-tolerant: force-killing the system will not lose data. To achieve
+this, the master writes edit logs of all metadata changes. On startup,
+a recovering master will read the edit logs to restore itself back to its previous state.
+We use the term "journal" to refer to the system of edit logs used to support fault-tolerance.
+The purpose of this doc is to help Alluxio administrators understand and manage the Alluxio journal.
 
 ## Configuration
 
 The most important configuration value to set for the journal is
-`alluxio.master.journal.folder`. This must be set to a shared filesystem available
-to all masters. In single-master mode, a local filesystem path is fine. With
-multiple masters distributed across different machines, the shared folder should
-be in a distributed system that supports flush such as HDFS or NFS. It is not
-recommended to put the journal in an object store. With an object store, every
-update to the journal requires a new object to be created, which is
+`alluxio.master.journal.folder`. This must be set to a filesystem folder that is
+available to all masters. In single-master mode, a local filesystem path may be
+used for simplicity. With
+multiple masters distributed across different machines, the folder must
+be in a distributed system where all masters can access it. The journal folder
+should be in a filesystem that supports flush such as HDFS or NFS. It is not
+recommended to put the journal in an object store like S3. With an object store, every
+metadata operation requires a new object to be created, which is
 prohibitively slow for most serious use cases.
 
 **Configuration examples:**
+
 Use HDFS to store the journal:
 ```
-alluxio.master.journal.folder=hdfs://[namenodeserver]:[namenodeport]/dir/alluxio_journal
+alluxio.master.journal.folder=hdfs://[namenodeserver]:[namenodeport]/alluxio_journal
 ```
 
-Use the local file system to store the journal: 
+Use the local file system to store the journal:
 ```
 alluxio.master.journal.folder=/opt/alluxio/journal
 ```
 
-## Formatting
+## Formatting the Journal
 
+Formatting the journal deletes all of its content and restores it to a fresh state.
 Before starting Alluxio for the first time, the journal must be formatted.
 
-**WARNING: formatting the journal will delete all Alluxio metadata**
 ```bash
-$ bin/alluxio formatMaster
+# This permanently deletes all Alluxio metadata, so be careful with this operation
+$ bin/alluxio formatMasters
```

Comment:
- `formatMasters` or `formatJournal`?
- `formatMasters` is not really shown up in `bin/alluxio`, looks like a secrete API

Reply:
changed back to `formatMaster`. In 2.0 we can update the CLI and start using `formatMasters`, which is more accurate